CMakeLists.txt

```diff
@@ -68,15 +68,16 @@ if(NOT foonathan_memory_FOUND)
     set(PATCH_COMMAND_STR PATCH_COMMAND
       git apply ${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.txt.patch &&
       sed -i -e "s/INSTALLER_PLATFORM/${INSTALLER_PLATFORM}/g" CMakeLists.txt)
-  elseif(BUILD_MEMORY_TESTS)
-    # This could be removed whenever we change the GIT_TAG from c619113 to anything above 09b884c
+  else()
+    # This could be removed whenever we change the GIT_TAG to anything above e9e13b0de82032051a49fd12ba2962e26949fb75
     set(PATCH_COMMAND_STR PATCH_COMMAND
-      curl https://github.com/foonathan/memory/commit/09b884c18abb314bfa678df27141a4dcad71c4ba.diff | git apply -)
+      curl https://github.com/MiguelCompany/memory/commit/6041b31341c323d07611051bb6e1970aabee5532.diff | git apply -)
```

Comment:
I see you are applying my PR foonathan/memory#119. I wonder if we should wait till that one is merged.

Reply:
Already in foonathan/memory `master` branch